packages/contentstack-import/src/import/modules/extensions.ts

```diff
@@ -209,4 +219,25 @@ export default class ImportExtensions extends BaseClass {
       }
     });
   }
+
+  getContentTypesInScope() {
+    const extension = values(this.extensions);
+    extension.forEach((ext: extensionType) => {
+      let ct: any = ext?.scope?.content_types || [];
+      if ((ct.length === 1 && ct[0] !== '$all') || ct?.length > 1) {
+        log(this.importConfig, `Removing the Content-types ${ct.join(',')} from Extension ${ext.title}`, 'success');
```

Comment:
@cs-raj , can we use info or warn instead of success as we're removing the content type from scope ?

Reply:
Done, used info